crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -24,6 +24,8 @@ const TOMLLIB_312_URL: &str = "https://raw.githubusercontent.com/python/cpython/
 // The "unresolved import" is because we don't understand `*` imports yet.
 static EXPECTED_DIAGNOSTICS: &[&str] = &[
     "/src/tomllib/_parser.py:7:29: Module 'collections.abc' has no member 'Iterable'",
+    "/src/tomllib/_parser.py:686:23: Object of type 'Unbound' is not callable",
```

Comment:
This feels like not-a-great error message for symbols with type `Unbound`... but we can probably hone that later. (But maybe we should add a TODO somewhere?)

Reply:
I feel like until we are actually making a concerted effort to polish up our diagnostics across red-knot, it's a little hard to know how we'd actually want to implement an improvement here, and thus where to even put the TODO. So I'm inclined to just leave this alone for now.

Reply:
(Especially considering I'm not 100% convinced that we'll even keep `Type::Unbound` around.)

Reply:
Fair enough! In general, usability is quite high priority for me, and I worry that this edge case might be harder to spot once our analysis improves (since I'm guessing the symbol in question here shouldn't have `Type::Unbound` to begin with). But I see your point.